flask_resty/view.py

```diff
@@ -18,6 +18,7 @@
 class ApiView(MethodView):
     schema = None
     id_fields = ('id',)
+    query_schema = None
```

Comment:
can we call it `querystring_schema` or `query_string_schema` to avoid semantic collision with sql query?

Reply:
In general, all the query things that talk about querystring. not a big deal

Reply:
maybe `args_schema`? not the clearest name, but consistent with what flask calls it otherwise. `query_string` looks clunky

Reply:
`qs_schema`?

Reply:
by analogy with https://docs.python.org/3/library/urllib.parse.html#urllib.parse.parse_qs

Reply:
json api calls it `parameters` :D

Reply:
“parameter”, for a single parameter, in the error, no?

Reply:
yeah, I mean, nobody knows how to call them... `args` or `qs` sound good